src/frontend/screens/MapScreen/index.tsx

```diff
@@ -32,18 +34,23 @@ const DEFAULT_ZOOM = 12;
 Mapbox.setAccessToken(config.mapboxAccessToken);
 const MIN_DISPLACEMENT = 3;
 
+function locationFilter({distance, time}: ParameterChanges) {
+  return time >= 1000 || distance >= 15;
```

Comment:
Note that this logic is different from what it was previously. The previous logic written for this function would be:

```
const MIN_TIME_INTERVAL = 200
const MAX_TIME_INTERVAL = 1000
const MIN_DISTANCE_INTERVAL = 1
const MAX_DISTANCE_INTERVAL = 15

// ...

return time > MIN_TIME_INTERVAL && (
  distance > MAX_DISTANCE_INTERVAL ||
  (time > MAX_TIME_INTERVAL && distance > MIN_DISTANCE_INTERVAL)
)
```

Reply:
@ErikSin - should implement this as suggested above